### PR TITLE
Updates Release 2.14

### PR DESCRIPTION
**Pull request by dsarkar**
- [x] Gulpfile 2.14.1
- [x] Privacy notice
- [x] Screenshots
- [x] FAQ
https://github.com/corona-warn-app/cwa-website/pull/2104
- [x] Blog
https://github.com/corona-warn-app/cwa-website/pull/2107

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -425,10 +425,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '15.1.1'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '2.13.2'))
+    .pipe(replace('[ios.current-app-version]', '2.14.1'))
     .pipe(replace('[android.latest-os-version]', '12'))
     .pipe(replace('[android.minimum-required-os-version]', '6'))
-    .pipe(replace('[android.current-app-version]', '2.13.3'))
+    .pipe(replace('[android.current-app-version]', '2.14.0'))
```
Comment (Ein-Tim) [inline gulpfile.js:431]:
```suggestion
    .pipe(replace('[android.current-app-version]', '2.14.1'))
```

See https://github.com/corona-warn-app/cwa-app-android/releases/tag/v2.14.1-RC1

**Review by Ein-Tim — COMMENTED** [review]
Bump version for Android?

**Comment by dsarkar** [comment]
@Ein-Tim Thanks

**Review by Ein-Tim — APPROVED** [review]
LGTM! Did not check screenshots.

**Pull request merged**